src/overlays/actors/ovl_En_Kaizoku/z_en_kaizoku.h

```diff
@@ -2,19 +2,98 @@
 #define Z_EN_KAIZOKU_H
 
 #include "global.h"
+#include "z64snap.h"
+#include "objects/object_kz/object_kz.h"
+
+#define KAIZOKU_GET_EXIT_INDEX(thisx) ((thisx)->params & 0x3F)
+#define KAIZOKU_GET_SWITCHFLAG(thisx) (((thisx)->params >> 6) & 0x7F)
 
 struct EnKaizoku;
 
 typedef void (*EnKaizokuActionFunc)(struct EnKaizoku*, PlayState*);
 
+typedef enum EnKaizokuAnimation {
```

Comment:
Any particular reason this needs to be in the header?

Reply:
I wanted to use the enum type for the `animIndex` member
```c
    /* 0x2E4 */ EnKaizokuAnimation animIndex;
```

Reply:
Could forward-declare it, doesn't seem useful to expose the actual values.

Reply:
Oh, I haven't thought on that, good idea

Reply:
welp, compiler doesn't like it
```
In file included from src/overlays/actors/ovl_En_Kaizoku/z_en_kaizoku.c:7:
src/overlays/actors/ovl_En_Kaizoku/z_en_kaizoku.h:45:36: error: field 'animIndex' has incomplete type
   45 |     /* 0x2E4 */ EnKaizokuAnimation animIndex;
      |                                    ^~~~~~~~~
make: *** [Makefile:354: build/src/overlays/actors/ovl_En_Kaizoku/z_en_kaizoku.o] Error 1
```

Reply:
Ah, apparently you can't forward-declare enums in C because the compiler is allowed to decide their width...